requirements.txt

```diff
@@ -1,3 +1,5 @@
-dbt==0.21.0
+dbt-core==1.0.3
+dbt-postgres==1.0.3
+dbt-snowflake==1.0.0
```

Comment:
That's a bit weird that the versions aren't staying synchronized any more, but probably makes some sense at the patch level.